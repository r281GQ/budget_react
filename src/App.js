import React, { Component } from "react";
import { Route } from "react-router";

import AboutComponent from "./components/about-component";
import HeaderComponent from "./components/header-component";
import GroupingComponent from "./components/grouping-component";
import CreateTransactionContainer from "./containers/create_transaction_container";
import AccountContainer from "./containers/account-container";
import TransactionListContainer from "./containers/transactin_list_container";
import BudgetContainer from "./containers/budget_container"
import AccountRoutes from './components/account_routes'

class App extends Component {
  render() {
    return (
      <div>
        <HeaderComponent />
        <Route path="/about" component={AboutComponent} />
        <Route path="/grouping" component={GroupingComponent} />
        <Route path="/new" component={CreateTransactionContainer} />
        <Route path="/transaction" component={TransactionListContainer} />
        <Route path="/account" component={AccountRoutes} />
        <Route path="/budget" component={BudgetContainer} />
      </div>
    );
  }
}

export default App;
