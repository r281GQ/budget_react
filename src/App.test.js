import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import App from './App';
import toJson from 'enzyme-to-json';
import Header from './components/header-component';
import Ac from './components/createAccountFormComponent';
const Label = () => <div>Element to test1</div>
it('renders without crashing', () => {
  // const div = document.createElement('div');
  // ReactDOM.render(<Label />, div);

  const wrapper = shallow(<Ac/>);

  expect(toJson(wrapper)).toMatchSnapshot();

});
