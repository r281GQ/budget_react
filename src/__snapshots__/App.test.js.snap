// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<form>
  <Field
    component="input"
    name="name"
    placeholder="name"
    type="text"
  />
  <Field
    component="input"
    name="initialBalance"
    placeholder="initialBalance"
    type="text"
  />
  <Field
    component="input"
    name="currency"
    placeholder="currency"
    type="text"
  />
  <Button
    active={false}
    block={false}
    bsClass="btn"
    bsStyle="default"
    disabled={false}
    type="submit"
  >
    Create
  </Button>
  <Button
    active={false}
    block={false}
    bsClass="btn"
    bsStyle="default"
    disabled={false}
  >
    Cancel
  </Button>
</form>
`;
